backend/main.go

```diff
@@ -24,11 +25,16 @@ var championPool = [5]int{29, 22, 18, 12, 10}
 // level is first array position + 1, tier is second array position + 1
 var dropRates [9][5]float64
 
+// list of drop rates by level; converted to culmulative rates per level
+// level is first array position + 1, tier is second array position + 1
+var dropRatesConv [9][5]float64
+
 // results counter
 type rollResult struct {
 	Champion    string
 	Cost        int
 	Appearances int
+	Percentage  string
```

Comment:
Use a `float32` or `float64` for percentages. Strings are not percentages and have no meaning.